client/src/containers/ProductDetails.jsx

```diff
@@ -37,7 +37,19 @@ const ProductDetails = ({itemId}) => {
     fetchData();
   }, [itemId]);
   useEffect(()=> {
-    setStyle(styles[0]);
+    var def = false;
+    for(let x of styles)
+    {
+      if(x["default?"]=== true) {
```

Comment:
Is this supposed to be styles[x]["default?"] as each key itself doesn't have a stored value inherently? Can you provide an example of the style object to clarify please?

Edit: noted it is a for...of loop so x stores the value at each key, would still like to validate that the key is labeled as "default?"